### PR TITLE
Update MySQL flexibleServer swagger to change some APIs

### PR DESCRIPTION
**Pull request by Suna-MS**
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Contribution checklist:
- [ ] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of “no breaking changes
- [ ] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [ ] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [x] Ensure to check this box if one of the following scenarios meet updates in the PR, so that label “WaitForARMFeedback” will be added automatically to involve ARM API Review. Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs, all “removals” and “adding a new property” no more require ARM API review.
  - Adding new API(s)
  - Adding a new API version
  - Adding a new service

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If there are following updates in the PR, ensure to request an approval from API Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 

- [ ] Removing API(s) in stable version
- [ ] Removing properties in stable version
- [ ] Removing API version(s) in stable version
- [ ] Updating API in stable version with Breaking Change Validation errors
- [ ] Updating API(s) in preview over 1 year

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

**Comment by ChenTanyi** [comment]
@Suna-MS Does the PR have minimal customer impact? Since it is a preview version.

**Comment by ChenTanyi** [comment]
Also, please fix the CI error

**Comment by Suna-MS** [comment]
@ChenTanyi 
There is no breaking changes in this PR, as this is  for a new service that in preview version and no customer impacted. 

Also the new APIs added in this PR are the same as  https://github.com/Azure/azure-rest-api-specs/pull/10586, which has  already been signed off from ARM side.

**Comment by ChenTanyi** [comment]
Got it

File: specification/mysql/resource-manager/Microsoft.DBforMySQL/preview/2020-07-01-privatepreview/examples/ServerKeyCreate.json [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        "kind": "azurekeyvault",
+        "properties": {
+          "serverKeyType": "AzureKeyVault",
+          "uri": "https://someVault.vault.azure.net/keys/someKey/01234567890123456789012345678901",
+          "creationDate": "2017-05-01T00:00:00.0Z"
+        }
+      }
+    },
+    "202": {
+      "body": {
```
Comment (majastrz) [inline specification/mysql/resource-manager/Microsoft.DBforMySQL/preview/2020-07-01-privatepreview/examples/ServerKeyCreate.json:30]: >body [](start = 7, length = 4)

This isn't actually a correct response for a 202. Please take a look at [https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/Addendum.md#asynchronous-operations](https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/Addendum.md#asynchronous-operations) for more information. (I may have missed this from the other Swagger.)

**Review by majastrz — APPROVED** [review]
Please fix the example 202 response as it's not correct. I signed off from ARM side because that doesn't really affect the shape of the API, but should be fixed to avoid misleading your users.

**Comment by Suna-MS** [comment]
Hi, @ChenTanyi 
Could you please help merge this PR? the swagger BreakingChange fail in CI is expected, as we will be in preview, no customer impact, so we have to do the change in the same file.

**Review by ChenTanyi — APPROVED** [review]

**Pull request merged**